modules/parrot_drush/manifests/components/php/drush_alias.pp

```diff
@@ -0,0 +1,25 @@
+define parrot_drush::components::php::drush_alias (
+  $root,
+  $ensure = 'present',
+  $uri = $name,
+  $php_executable = undef,
+  $protocol = '',
+)
+{
+
+  file {"/home/vagrant/.drush/$name.aliases.drushrc.php":
+    ensure => $ensure,
+    owner => 'root',
+    group => 'root',
+    content => template('parrot_drush/php/drush_alias.erb'),
+    #require => File['/home/vagrant/.drush'],
```

Comment:
I commented out the two require statements here, because they failed for me. Do we really need to require them?

Reply:
Yeah, you'll need them, in case the `'/home/vagrant/.drush'` directory doesn't exist.
This will need to be declared created somewhere else in the Puppet manifests though.

Reply:
(now actually fixed, oops!)